.github/workflows/release.yaml

```diff
@@ -82,7 +87,10 @@ jobs:
         run: podman login -u ${{ secrets.BUNDLE_PUSH_USER  }} -p ${{ secrets.BUNDLE_PUSH_PASS }} quay.io
 
       - name: Create image from snapshot
-        run: make push-snapshot-image IMAGE_TAG=${{ github.sha }}
+        run: make push-snapshot-image IMAGE_TAG=$TAG IMAGE_REPO=$IMAGE_REPO
+
+      - name: Create and push the tekton bundle tagged with sha
+        run: make task-bundle TASK_TAG=$TAG TASK=<( yq e '.spec.steps[].image? = "$IMAGE_REPO:$TAG"' $TASK)
```

Comment:
I think embedding an yq expression in this file will make it kind of hard to debug things. 

I'm also surprised that this works: `'.spec.steps[].image? = "$IMAGE_REPO:$TAG"'` 
I would've expected the variable replacement to not occur because of the outer single quotes.